cross/arm/toolchain.cmake

```diff
@@ -4,14 +4,16 @@ set(CMAKE_SYSTEM_NAME Linux)
 set(CMAKE_SYSTEM_VERSION 1)
 set(CMAKE_SYSTEM_PROCESSOR armv7l)
 
+set(TOOLCHAIN "arm-linux-gnueabihf")
+
 add_compile_options(-target armv7-linux-gnueabihf)
```

Comment:
Toolchain name is '**arm**-linux-gnueabihf' while target name is '**armv7**-linux-gnueabihf'. So they are different.